File: editor/en/[217]Contains Duplicate.java
//leetcode submit region begin(Prohibit modification and deletion)
class Solution {
  public boolean containsDuplicate(int[] nums) {
    Set<Integer> set = new HashSet<>();
    for (int num : nums) {
      if (!set.add(num)) {
        return true;
      }
    }
    return false;
  }
}
// leetcode submit region end(Prohibit modification and deletion)

File: editor/en/[217]Contains Duplicate.js
//leetcode submit region begin(Prohibit modification and deletion)
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function (nums) {
    let set = new Set(nums);
    return nums.length !== set.size;
};
//leetcode submit region end(Prohibit modification and deletion)
